Concrete/Magento2CoreSetup.php

```diff
@@ -438,17 +438,17 @@ static private function getBrandConfig($storeConfig, $section)
         $selectedBrands = $storeConfig->getValue($section .  'cctypes', $scope, $storeId) ?? "";
         $brands = array_merge([''], explode(
             ',',
-            $selectedBrands
+            $selectedBrands ?? ''
```

Comment:
This condition is already handled in line 438

Reply:
Reversed! Thanks!